packages/upswyng-server/src/components/ResourceEditor.svelte

```diff
@@ -80,7 +80,7 @@
     {/if}
     {#if resource.resourceId}
       <p>
-        <span class="label">Resource ID</span>
+        <span class="label">Provider ID</span>
```

Comment:
I think we need to make sure all occurrences say "Service Provider"

Reply:
👍 [updated](https://github.com/CodeForBoulder/upswyng/pull/333/commits/8fac2ab788a6f081cde6ab8535f499a42038d3ae)